qemu/tests/cfg/block_hotplug_negative.cfg

```diff
@@ -0,0 +1,69 @@
+# Below case works stablly on QMP monitor, but have no more test with Human
+# monitor, so suggest to use QMP monitor as default qemu monitor run below test
+#
+- block_hotplug_negative:
+    no RHEL.3.9
+    no ide
+    virt_test_type = qemu
+    type = block_hotplug_negative
+    bootindex_image1 = 0
+    images += " stg0"
+    boot_drive_stg0 = no
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 1G
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+
+    variants:
+        - one_pci:
+            blk_num = 1
+
+    variants:
+        - fmt_qcow2:
+            image_extra_params = ""
+            img_format_type = qcow2
+            image_format_stg0 = qcow2
+            image_format_stg1 = qcow2
+        - fmt_raw:
+            image_extra_params = ""
+            img_format_type = raw
+            image_format_stg0 = raw
+            image_format_stg1 = raw
+
+    variants:
+        - block_virtio:
+            pci_type = virtio-blk-pci
+            drive_format_stg0 = virtio
+            drive_format_stg1 = virtio
+            get_disk_pattern = "^/dev/vd[a-z]*$"
+            virtio_blk:
+                no ppc64 ppc64le
+        - block_scsi:
+            pci_type = scsi-hd
+            drive_format_stg0 = scsi-hd
+            drive_format_stg1 = scsi-hd
+            get_disk_pattern = "^/dev/sd[a-z]*$"
+            virtio_blk:
+                need_controller = yes
+                controller_model = virtio-scsi-pci
+
+    variants:
+        - @default:
+            only block_virtio.fmt_raw.one_pci
+            block_virtio:
+                blk_extra_params_stg0 = "x_data_plane=on"
+            block_scsi:
+                bus_extra_params_stg0 = "x_data_plane=on" 
+        - data_plane:
+            no Host_RHEL.m6
+            iothreads = iothread0
+            virtio_blk:
+                blk_extra_params_image1 = "iothread=${iothreads}"
+            virtio_scsi:
+                no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
+                bus_extra_params_image1 = "iothread=${iothreads}"
+            block_virtio:
+                blk_extra_params_stg0 = "x_data_plane=on"
+            block_scsi:
+                bus_extra_params_stg0 = "x_data_plane=on"
```

Comment:
line 54~57 are same as line 66~69, please put it in the general part.

Reply:
have updated it, thanks